.github/workflows/tox.yml

```diff
@@ -140,11 +204,7 @@ jobs:
       run: |
         echo 'local_scheme = "no-local-version"' >> pyproject.toml
         git update-index --assume-unchanged pyproject.toml
-    - name: Pre-populate tox env
-      run: python -m tox -p auto --parallel-live -vvvv --notest
-    - name: Build dists
-      run: python -m tox -p auto --parallel-live -vvvv
```

Comment:
Why did you remove building and checking dists? No nothing will be built and published.